reference/customers.v3.yml

```
@@ -1556,6 +1556,47 @@ paths:
                   - customer_id: 12
                     name: What is your favorite pizza topping?
                     value: Mushrooms
+    post:
+      responses:
+        '200':
+          $ref: '#/components/responses/FormFieldValuesResponse'
+        '422':
+          description: 'The form field value was not valid. This is the result of missing required fields, or of invalid data. See the response for more details.'
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  status:
+                    type: integer
+                    description: The HTTP status code.
+                    format: int32
+                  title:
+                    type: string
+                    description: The error title describing the particular error.
+                  type:
+                    type: string
+                  instance:
+                    type: string
+                  errors:
+                    type: object
+      description: Creates a list of form field values for the Customer or Customer Address object.
+      summary: Create Customer Form Field Values
+      operationId: post-customers-form-field-values
```

```suggestion
      operationId: postCustomersFormFieldValues
```
I would recommend using camelcase for `operationId`